### ISSUE TITLE
Relink config tiddlers that contain single titles as text

### ISSUE THREAD

**Issue by Mythos-Writer** [flibbles/tw5-relink#33]
Hello,

I use the Relink plugin heavily, and I've been writing a plugin which integrates with it that uses config tiddlers, like ``$:/config/my/plugin/defaultTiddler`` which will contain a single tiddler as a text value. Is there a way to make that config tiddler update if the tiddler named inside it is changed?

I tried looking through the code myself, and it looks like I could do this with the `relinktext` module type. The `text.js` one looks like it already makes an exception for $:/DefaultTiddlers to treat it like a filter, and there's that comment which says to talk to you about making more?

Thanks in advance. You made an amazing plugin.

Mythos

**Comment by Mythos-Writer** [comment]
Yes, I think that solution would meet my needs just fine. Thank you very much!

Mythos

**Comment by flibbles** [comment]
Are you concerned with maintaining backward compatibility with TiddlyWiki before 5.1.23? Cause if not, you use the inputActions attribute. Just have your edit-text widget have something like this:
```html
<$edit-text tiddler='$:/config/my/plugin/defaultTiddler' inputActions="""<$action-setfield $tiddler='$:/config/my/plugin/defaultTiddler' type='text/x-tiddler-title' />""" />
```
This should make it that when the config tiddler is generated, it will automatically get the correct tiddler type, and you wouldn't have to be a shadow tiddler.

**Comment by Mythos-Writer** [comment]
Oh! Actually, that works quite well. Yes, this solution works for me again.
